cylc/flow/config.py

```diff
@@ -1332,7 +1332,7 @@ def configure_sim_modes(self):
             # TODO - to use batch schedulers in dummy mode we need to
             # identify which resource directives to disable or modify.
             # (Only execution time limit is automatic at the moment.)
-            rtc['job']['batch system'] = 'background'
+            rtc['platform'] = 'localhost'
```

Comment:
Technically this breaks dummy mode, but we've already agreed in principle that dummy-mode only serves to test network connections, and should go.